### PR TITLE
Celery task to clear schedule was added

### PR DESCRIPTION
**Pull request by aidarbek**
## What type of PR is this? (check all applicable)

- [x] Feature

## Related Tickets & Documents
Solves #3375 issue

### CODE REVIEW

**Comment by aidarbek** [comment]
@rauchy that perfectly makes sense, thanks. I've made some changes, hope everything is good now once all checks are passed

**Comment by aidarbek** [comment]
There are some visual changes that seem unrelated

File: redash/worker.py [hunk 1]
```diff
@@ -22,6 +22,10 @@
         'task': 'redash.tasks.refresh_queries',
         'schedule': timedelta(seconds=30)
     },
+    'empty_schedules': {
+        'task': 'redash.tasks.empty_schedules',
+        'schedule': timedelta(seconds=30)
```
Comment (rauchy) [inline redash/worker.py]: IMO this task can run at a much lower frequency. (say, once an hour)

File: redash/models/__init__.py [hunk 2]
```diff
@@ -549,6 +549,23 @@ def by_user(cls, user):
...
+    @classmethod
+    def past_scheduled_queries(cls):
+        now = utils.utcnow()
+        queries = (
+            Query.query
+            .options(joinedload(Query.latest_query_data).load_only('retrieved_at'))
+            .filter(Query.schedule.isnot(None))
+            .order_by(Query.id)
+        )
+        past_scheduled_queries = []
```
Comment (rauchy) [inline redash/models/__init__.py]: Does it feel like a comprehension or filter would be more apt here? Not sure.

File: tests/tasks/test_refresh_queries.py [hunk 3]
```diff
@@ -65,3 +67,11 @@ def test_enqueues_parameterized_queries(self):
             add_job_mock.assert_called_with(
                 "select 42", query.data_source, query.user_id,
                 scheduled_query=query, metadata=ANY)
+
+    def test_empty_schedules(self):
```
Comment (rauchy) [inline tests/tasks/test_refresh_queries.py]: This guy deserves his own test class too :)

**Comment by aidarbek** [comment]
Some visual changes yet again :)

**Comment by gabrieldutra** [comment]
> Some visual changes yet again :)

Just sync with master and they'll be gone :slightly_smiling_face:

File: redash/models/__init__.py (hunk 2 above)
Comment (aidarbek) [reply redash/models/__init__.py]: Used filter here. Also, added another unit test for this method specifically

**Comment by aidarbek** [comment]
Hi @rauchy, could you check my latest changes, please?

File: redash/tasks/queries.py [hunk 4]
```diff
@@ -159,6 +159,14 @@ def enqueue_query(query, data_source, user_id, is_api_key=False, scheduled_query
     return job
 
 
+@celery.task(name="redash.tasks.empty_schedules")
+def empty_schedules():
+    queries = models.Query.past_scheduled_queries()
+    for query in queries:
+        query.schedule = None
```
Comment (arikfr) [inline redash/tasks/queries.py]: Let's add a log message here, just so admins will be able to traceback changes.

File: tests/test_models.py [hunk 5]
```diff
@@ -282,6 +282,20 @@ def test_schedule_until_before(self):
...
+        Query.past_scheduled_queries() before the given time is past.
+        """
+        one_day_ago = (utcnow() - datetime.timedelta(days=1)).strftime("%Y-%m-%d")
+        two_hours_ago = utcnow() - datetime.timedelta(hours=2)
+        query = self.factory.create_query(schedule={'interval':'3600', 'until':one_day_ago, 'time': None, 'day_of_week':None})
+        query_result = self.factory.create_query_result(query=query.query_text, retrieved_at=two_hours_ago)
+        query.latest_query_data = query_result
+
+        queries = models.Query.past_scheduled_queries()
+        self.assertIn(query, queries)
```
Comment (arikfr) [inline tests/test_models.py]: This feels very similar to `QueryTest.test_past_scheduled_queries` -- why two tests?

Comment (aidarbek) [reply tests/test_models.py]: I just realised that QueryTest would be more appropriate place, but forgot to delete it from here :P

**Comment by arikfr** [comment]
Thanks, @aidarbek ! :)

**Pull request merged**